assets/js/float/curvedLines.js

```
@@ -0,0 +1,483 @@
+			}
+
+			return result;
+		}
+		
+		function hasInvalidParameters(curvedLinesOptions) {
+			if (typeof curvedLinesOptions.fit != 'undefined' ||
+			    typeof curvedLinesOptions.curvePointFactor != 'undefined' ||
+			    typeof curvedLinesOptions.fitPointDist != 'undefined') {
+			    	throw new Error("CurvedLines detected illegal parameters. The CurvedLines API changed with version 1.0.0 please check the options object.");
+			    	return true;
+			    }
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)